cirq/testing/consistent_pauli_expansion_test.py

```diff
@@ -39,9 +39,7 @@ def _unitary_(self) -> np.ndarray:
         return np.eye(2)
 
 
-class GoodGateNoPauliExpansion(cirq.MultiQubitGate):
-    def __init__(self) -> None:
-        super().__init__(num_qubits=4)
+class GoodGateNoPauliExpansion(cirq.FixedQubitCountGate(4)):
```

Comment:
this is the "cool" part about this construct - it's kind of a dependent type implementation. However we might want to generate and "cache" the first 10 types upfront - I'm not sure about the cost of creating the same type over and over again.